docker_scripts/start.sh

```diff
@@ -1 +1,3 @@
-docker run -t -i -v /var/run/docker.sock:/var/run/docker.sock -p 3000:3000 -p 4000:4000 gcperkins/axon:latest
\ No newline at end of file
+xdg-open http://localhost:3000
```

Comment:
Does this work with WSL?

Reply:
Unsure. I can go try. Working on ubuntu 20.04 right now

Reply:
I will be making another start script for windows